cartography/util.py

```diff
@@ -133,6 +135,26 @@ def timed(*args, **kwargs):  # type: ignore
     return cast(F, timed)
 
 
+def aws_paginate(
+    client: boto3.client,
+    method_name: str,
+    object_name: str,
+    **kwargs: Any,
+) -> List[Dict]:
+    '''
+    Helper method for boilerplate boto3 pagination
+    The **kwargs will be forwarded to the paginator
+    '''
+    paginator = client.get_paginator(method_name)
+    items = []
+    i = 0
+    for i, page in enumerate(paginator.paginate(**kwargs), start=1):
+        if i % 100 == 0:
+            logger.info(f'fetching page number {i}')
```

Comment:
100 is the default page size used by paginator, could you pass that configuration specifically?
From: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/inspector.html#paginators
Something like :
```
paginator.paginate(**kwargs, PaginationConfig={
        'PageSize': 100
    }
```

Reply:
Specifically for list_findings, it is 100. Other apis will have other defaults. And it's possible that some api's will error on a page size as high as 100.

It's my intention that this new `aws_pagiante` function be used for other sync modules, so I do not want to force the user to need to be aware of an optimal page size.